pipeline/cactus_workflow.py

```diff
@@ -352,16 +393,30 @@ def run(self):
                                                        trimMinSize=self.getOptionalPhaseAttrib("trimMinSize", int, 0),
                                                        trimThreshold=self.getOptionalPhaseAttrib("trimThreshold", float, 0.8),
                                                        trimWindowSize=self.getOptionalPhaseAttrib("trimWindowSize", int, 10),
-                                                       trimOutgroupFlanking=self.getOptionalPhaseAttrib("trimOutgroupFlanking", int, 100)), ingroups, outgroups, alignmentsFile, outgroupsDir))
+                                                       trimOutgroupFlanking=self.getOptionalPhaseAttrib("trimOutgroupFlanking", int, 100)), ingroupIDs, outgroupIDs)).rv()
+        self.phaseNode.attrib["trimBlastResults"] = trimBlastResults
+
+        return self.makeFollowOnPhaseJob(CactusTrimmingBlastPhase2, phaseName="trimBlast")
+class CactusTrimmingBlastPhase2(CactusPhasesJob):
+    def run(self, fileStore):
+        exp = ExperimentWrapper(self.cactusWorkflowArguments.experimentNode)
+        seqIDs = self.phaseNode.attrib["seqIDs"].split()
+        seqMap = exp.buildSequenceMap()
+        seqIDMap = dict(zip(seqMap.keys(), seqIDs))
+        trimBlastResults = self.phaseNode.attrib["trimBlastResults"]
+        
+        outgroupFragmentIDs = trimBlastResults["outgroupFragmentIDs"]
+        alignmentsID = trimBlastResults["alignmentsID"]
+
+        self.phaseNode.attrib["aignmentsID"] = alignmentsID
```

Comment:
Do these phaseNodes get properly carried over without a shared filesystem? I think this may be one of the places where there is an assumption that saving/reloading the config or experiment XML works. Maybe this works fine, though?